docs/manual/manual.adoc

```diff
@@ -1120,6 +1120,45 @@ For example, to generate a blueprint remediation for RHEL 8 OSPP profile, run:
 $ oscap xccdf generate fix --profile ospp --fix-type blueprint /usr/share/xml/scap/ssg/content/ssg-rhel8-ds.xml > blueprint.toml
 ----
 
+=== Generating RHEL Kickstarts
+
+OpenSCAP can generate RHEL kickstarts which can be used for unattended installation of RHEL, Fedora and similar systems.
+Information about RHEL kickstarts and their syntax can be found at https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/performing_an_advanced_rhel_9_installation/kickstart_references[Kickstart references].
+
+To generate a kickstart, use `oscap xccdf generate fix` command with the `--fix-type kickstart` option.
+
+The kickstart will be generated from kickstart snippets in XCCDF rules in the input SCAP content.
+The kickstart snippets need to be stored in `<fix>` elements with `system` attribute set to `urn:xccdf:fix:script:kickstart`.
+
+When processing the kickstart snippets comming from the XCCDF Rules, each line is processed separately.
+The following rules are applied on each line:
+
+* lines starting with `#` are ignored
+* empty lines are ignored
+* lines starting with a supported command are processed
+* lines starting with something else than a supported command are dropped
+* excess whitespace are trimmed
+
+Supported commands:
+* `package install package_name` - adds `package_name` to `%packages` section in the kickstart
+* `package remove package_name` - adds `-package_name` to `%packages` section in the kickstart
+* `service enable service_name` - adds `service_name` to list in the `--enabled=` option in the `services` command in commands section in the kickstart
+* `service disable service_name` - adds `service_name` to list in the `--disabled=` option in the `services` command in commands section in the kickstart
+* `post command` - adds `command` to the `%post` section the kickstart
+* `logvol command` - adds `logvol command` to the commands section the kickstart
```

Comment:
I think this needs to be split into specific fields for a better defined experience.